app/views/pages/privacy.html.erb

```diff
@@ -1,22 +1,103 @@
 <% content_for :title, "Privacy policy" %>
 
 <div class="text">
-  <h1 class="heading-large">Privacy policy</h1>
-  <p>The waste exemptions service is run by the Environment Agency. Information you provide is handled in accordance with its <a rel="external" href="https://www.gov.uk/government/organisations/environment-agency/about/personal-information-charter">personal information charter.</a></p>
-  <p>The personal information you provide will be used to update the public register for exempt waste operations.</p>
-  <p>This will include:</p>
+  <h1 class="heading-large">
+    Privacy Policy: how we use your personal information 
+  </h1>
+  <p>
+    We are the Environment Agency and we run the Register Your Waste Exemptions service. We are the 
+    data controller for this service. A data controller determines how and why personal data 
+    (personal information) is processed.
+  </p>
+  <p>
+    Our <a rel="external" href="https://www.gov.uk/government/organisations/environment-agency/about/personal-information-charter">personal information charter.</a> 
```

Comment:
There's a stray `.` at the end of the link text.

Reply:
Well caught. I'll fix this momentarily.

Reply:
This has been fixed in the latest commit.